docs/virtualkeyboard/index.html

```diff
@@ -0,0 +1,455 @@
+            <dt>
+                <dfn>hide()</dfn> method
+            </dt>
+              <dd>
+                <p>
+                  The method must follow these steps:
+                </p>
+                <ol>
+                  <li>If the <a>document</a> doesn't have <a href="https://html.spec.whatwg.org/multipage/interaction.html#sticky-activation">sticky user activation</a>, abort these steps.
+                  </li>
+                  <li>Trigger {{VirtualKeyboard/hide()}}call that invokes a system API to hide the keyboard.
+                  </li>
+                </ol>
+               </dd>
+               <p class="note">
+                Platform heuristics may impose additional restrictions on VK {{VirtualKeyboard/show()}} and {{VirtualKeyboard/hide()}}.
+              </p>
+              <p class="note">
+                A few people have expressed concerns around {{VirtualKeyboard/show()}} and {{VirtualKeyboard/hide()}} methods not being promise-based. However, we couldn't find any real use cases around needing promises in this type of operation.
+              </p>
+            <dt>
+              <dfn>overlaysContent</dfn>
+            </dt>
+            <dd>
+              <p>
+                The attribute enables the author to opt-out of resizing the visual viewport.
+              </p>
+            </dd>
+            <dt>
+              <dfn>boundingRect</dfn>
+            </dt>
+            <dd>
+            <p>
+                The attribute reports the intersection of the VK with the layout viewport in client coordinates.
+            </p>
+            </dd>
+            <dt>
+                <dfn>ongeometrychange</dfn>
+              </dt>
+              <dd>
+              <p>
+                  The event is dispatched when the intersection of the VK and the layout viewport changes, e.g. in response to the VK being shown or hidden or the browser window being repositioned.
+              </p>
+              </dd>
+        </dl>
+    </section>
+    <section data-dfn-for="ElementContentEditable">
+        <h2>Extensions to the <dfn>ElementContentEditable</dfn> mixin</h2>
+        <pre class="idl">
+           partial interface ElementContentEditable {
+            [CEReactions, Reflect, ReflectOnly=("auto","manual")] 
+            attribute DOMString virtualKeyboardPolicy;
+           };
+        </pre>
+        <p>The <dfn><code>virtualKeyboardPolicy</code></dfn> is an <a href=https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#enumerated-attribute>enumerated attribute</a> whose keywords are the empty string, auto, and manual.
+        <p>When specified on an element that is a contenteditable host, <dfn><code>auto</code></dfn> causes the corresponding editable element to automatically show the VK when it is focused or tapped & <dfn><code>manual</code></dfn> 
+            decouples focus and tap on the editable element from changes in the VK’s current state - the VK remains as it was.
```

Comment:
Similarly to the above, you probably want to define the processing for this attribute: what happens when it's set, changed, removed, etc.

Reply:
updated